File: app/src/main/java/com/screenmeet/sdkdemo/WebViewActivity.java
package com.screenmeet.sdkdemo;

import android.os.Bundle;
import android.webkit.WebView;
import android.webkit.WebViewClient;

import androidx.appcompat.app.AppCompatActivity;

import com.screenmeet.sdk.ScreenMeet;

public class WebViewActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_single_webview);

        WebView webView = findViewById(R.id.wv);

        webView.getSettings().setSupportZoom(true);
        webView.getSettings().setBuiltInZoomControls(true);
        webView.getSettings().setJavaScriptEnabled(true);
        webView.setWebViewClient(new WebViewClient());

        //For confidentiality example - navigate to Login page
        webView.loadUrl("https://cbdemo.screenmeet.com/");

        ScreenMeet.appStreamVideoSource().setConfidential(webView, true);
    }
}

File: android/app-kotlin/src/main/java/com/screenmeet/sdkkotlindemo/WebViewActivity.kt
package com.screenmeet.sdkkotlindemo

import android.os.Bundle
import android.webkit.WebView
import android.webkit.WebViewClient
import androidx.appcompat.app.AppCompatActivity
import com.screenmeet.sdk.ScreenMeet

class WebViewActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_single_webview)

        val webView = findViewById<WebView>(R.id.wv)
        webView.settings.setSupportZoom(true)
        webView.settings.builtInZoomControls = true
        webView.settings.javaScriptEnabled = true
        webView.webViewClient = WebViewClient()

        //For confidentiality example - navigate to Login page
        webView.loadUrl("https://cbdemo.screenmeet.com/")
        ScreenMeet.appStreamVideoSource().setConfidential(webView, true)
    }
}